crates/cairo-coverage/src/input/data_loader/types.rs

```diff
@@ -0,0 +1,37 @@
+use crate::types::{FileLocation, FunctionName, LineNumber};
+use cairo_lang_sierra::program::StatementIdx;
+use serde::Deserialize;
+use std::collections::HashMap;
+use std::ops::RangeInclusive;
+
+pub type CodeLocation = (FileLocation, LineRange);
+
+#[derive(Deserialize, Clone, Eq, PartialEq)]
+pub struct LineRange {
+    pub start: Position,
+    pub end: Position,
+}
+
+impl IntoIterator for &LineRange {
+    type Item = LineNumber;
+    type IntoIter = RangeInclusive<LineNumber>;
```

Comment:
Yep, 

```json
              {
                "end": {
                  "col": 4,
                  "line": 3
                },
                "start": {
                  "col": 4,
                  "line": 3
                }
              }
```

This is needs to be inclusive otherwise it doesn't make sense

Reply:
https://github.com/software-mansion/cairo-coverage/issues/27